CODEOWNERS

```
@@ -31,7 +31,19 @@
 /vault/plugin_catalog.go      @hashicorp/vault-ecosystem
 
 # Content on developer.hashicorp.com
-/website/ @hashicorp/vault-education-approvers
+
+# Engineering and web presence get notified of, and can approve changes to web tooling, but not content.
+
+/website/               @hashicorp/web-presence @hashicorp/vault
+/website/data/
+/website/public/
+/website/content/
+
+# Education and engineering get notified of, and can approve changes to web content.
+
+/website/data/          @hashicorp/vault-education-approvers @hashicorp/vault
+/website/public/        @hashicorp/vault-education-approvers @hashicorp/vault
+/website/content/       @hashicorp/vault-education-approvers @hashicorp/vault
```

```suggestion
/website/data/          @hashicorp/vault-education-approvers @hashicorp/vault
/website/public/        @hashicorp/vault-education-approvers @hashicorp/vault
/website/content/       @hashicorp/vault-education-approvers @hashicorp/vault
/website/templates/          @hashicorp/vault-education-approvers @hashicorp/vault
```
I'm leaving the `@hashicorp/vault` references for now, but I don't think it's appropriate. If specific teams what notifications for docs related to their work, I understand that, but making all of Vault owners for the docs will create a lot of extra GitHub noise and also allow folks to bypass EDU review for doc changes.